src/rpc/misc.cpp

```diff
@@ -949,6 +949,172 @@ UniValue getaddressbalance(const JSONRPCRequest& request)
 
 }
 
+UniValue getanonymityset(const JSONRPCRequest& request)
```

Comment:
Access to sigma state should be protected by `cs_main`. Applies to all RPC functions introduced in this PR

Reply:
Thanks, fixed